### PR TITLE
Integrate DATS validator in Travis tests

### PR DESCRIPTION
**Pull request by joeyzhou98**
## Description
- TravisCI tests now run the DATS validator on all DATS.json files in each submodule under directories "projects" and "investigators"
- Need to fix all DATS files before merging this PR so that the build stays passing

### CODE REVIEW

**Comment by xlecours** [comment]
This seems to point out errors in our DATS files. Which is a good thing because there are not valid according to the schemas.

File: scripts/crawl_zenodo.sh [hunk 1]
```diff
@@ -19,5 +19,5 @@ touch ${TOUCHFILE}
 
 (cd ${BASEDIR} && git pull main master)
 
-docker run --rm -u $UID:$UID -v $HOME:$HOME -e HOME=$HOME -v ${BASEDIR}:/workdir -w /workdir bigdatalabteam/git-annex python3 ./scripts/crawl_zenodo.py --force --verbose -z ${TOKEN_LIST} &>>${LOGFILE}
+docker run --rm -u $UID:$UID -v $HOME:$HOME -e HOME=$HOME -v ${BASEDIR}:/workdir -w /workdir bigdatalabteam/git-annex python3 ./scripts/crawl_zenodo.py --force --verbose -z ${TOKEN_LIST} &>>${LOGFILE} || \rm ${TOUCHFILE}
```
Comment (dlq) [inline scripts/crawl_zenodo.sh]: I'm unsure how adding `|| \rm ${TOUCHFILE}` does anything since the next line is `\rm ${TOUCHFILE}`

Comment (joeyzhou98) [reply scripts/crawl_zenodo.sh]: You are right, let me remove it

**Comment by glatard** [comment]
As discussed today:
* @joeyzhou98 will fix the DATS model for the PERFORM dataset
* @emmetaobrien or @xlecours will fix the Prevent-AD and possibly OpenPain ones and PR to `joeyzhou98:integrate_dats_validator`

**Pull request merged**